uzupelnianka.txt

```diff
@@ -1,20 +1,20 @@
-Klasa to …
+Klasa to "szablon" do tworzenia obiektów
 
-Obiekt to …
+Obiekt to instancja utworzona na podstawie klasy
 
-Zmienna to …
+Zmienna to składowa klasy przechowująca wartości proste lub referencje do obiektów
```

Comment:
Zmienna to niekoniecznie składowa klasy, zmienne mogą być lokalne dla metod, albo konstruktorów. "Zmienne" na poziomie klasy nazywamy najczęściej polami